test/framework/module_generator.py

```diff
@@ -249,8 +249,8 @@ def test_load_msg(self):
             tcl_load_msg = '\n'.join([
                 '',
                 "if { [ module-info mode load ] } {",
-                "    puts stderr \"test \\$test \\$test",
-                "test \\$foo \\$bar\"",
+                "    puts stderr \"\"\"test \\$test \\$test",
+                "test \\$foo \\$bar\"\"\"",
```

Comment:
hmm, is this what we want? why do quotes get wrapped in quotes now?

it seems like `to_str` differs from `quote_str` in this?